src/service/service.rs

```diff
@@ -26,6 +26,11 @@ pub trait Service {
     /// The `Future` returned by this `Service`.
     type Future: Future<Item=Response<Self::ResBody>, Error=Self::Error>;
 
+    #[doc(hidden)]
+    fn poll_ready(&mut self) -> Poll<(), Self::Error> {
```

Comment:
This should probably not be `doc(hidden)`, but include similar wording to [what it says in tower-service](https://docs.rs/tower-service/0.2.0/tower_service/trait.Service.html#tymethod.poll_ready).

Reply:
I've added doc comment to them.